### DISCUSSION TITLE
Augmentors

### DISCUSSION THREAD

**Discussion by greenhawktobias** [azavea/raster-vision#1673] [Q&A] [answered]
I am running a script for semantic segmentation without augmentors defined in the code, `augment: bool = False` But when running the script in cmd it says `augmentors=['RandomRotate90', 'HorizontalFlip', 'VerticalFlip']`. Are those basic augmentors always present?

**Comment by AdeelH** [comment, marked as answer]
Yes, those basic augmentations are used by default but can be disabled by passing `augmentors=[]` to `SemanticSegmentationGeoDataConfig`/`SemanticSegmentationImageDataConfig`. See docs for `augmentors` and `aug_transform` in [`DataConfig`](https://docs.rastervision.io/en/0.20/api_reference/_generated/rastervision.pytorch_learner.learner_config.DataConfig.html#dataconfig).
